lib/yearbook_web/templates/layout/_header.html.heex

```diff
@@ -23,10 +26,12 @@
       </div>
     </div>
     <div class="flex justify-center mt-4 space-x-6 lg:hidden">
-      <%= for page <- navbar(assigns) do %>
-        <a href={page.path} class="text-base font-medium text-white hover:text-orange-50">
-          <%= page.title %>
-        </a>
+      <%=if Enum.any?(@current_user.permissions, &(&1 == :admin)) do %>
```

Comment:
We should probably instead of showing the `if` statement, we should change what `navbar(assigns)` returns.